File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/03. Film Premiere/03. Film Premiere.js
function filmPremiere(projection, packet_type, ticket_count) {
    const movie_dict = {
        'John Wick' : {'Drink' : 12, 'Popcorn' : 15,'Menu': 19 },
        'Star Wars' : {'Drink' : 18, 'Popcorn' : 25,'Menu': 30 },
        'Jumanji'   : {'Drink' : 9, 'Popcorn' : 11,'Menu': 14 },
    }

    let total_sum = movie_dict[projection][packet_type] * ticket_count

    if (projection === 'Star Wars' && ticket_count >= 4) {
        total_sum *= 0.70
    } else if (projection === 'Jumanji' && ticket_count == 2) {
        total_sum *= 0.85
    }

    return `Your bill is ${total_sum.toFixed(2)} leva.`
}

//console.log(filmPremiere('John Wick', 'Drink', 6))

File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/03. Film Premiere/03. Film Premiere.py
projection = input()
packet_type = input()
ticket_count = int(input())

movie_dict = {
    'John Wick' : {'Drink' : 12, 'Popcorn' : 15,'Menu': 19 },
    'Star Wars' : {'Drink' : 18, 'Popcorn' : 25,'Menu': 30 },
    'Jumanji'   : {'Drink' : 9, 'Popcorn' : 11,'Menu': 14 },
}

total_sum = movie_dict[projection][packet_type] * ticket_count

if projection == 'Star Wars' and ticket_count >= 4:
    total_sum *= 0.70
elif projection == 'Jumanji' and ticket_count == 2:
    total_sum *= 0.85

print(f'Your bill is {total_sum:.2f} leva.')
